workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/checker/TransformChecker.java

```diff
@@ -56,7 +62,8 @@ public WorkReport checkWorkFlowStatus(WorkContext workContext) {
 						new IllegalArgumentException("Cannot get the project transformation output from the list"));
 			}
 			if (!Objects.equals(output.getStatus(), "done")) {
-				return new DefaultWorkReport(WorkStatus.FAILED, workContext);
+				return new DefaultWorkReport(WorkStatus.FAILED, workContext,
+						getMessage(server, workspaceID, projectID, transformID));
```

Comment:
the `server` is actually `this.server` it can be used from within `getMessage()` without a need to pass it as an argument.